test/jot_test.gleam

```diff
@@ -1,29 +1,31 @@
-import gleam/io
 import gleam/list
-import gleam/string
-import gleeunit
+import gleam/int
 import jot
 import jot_test/support
+import testbldr
+import testbldr/pieces as testpieces
```

Comment:
Maybe this isn't to discuss here but how come this second module is needed? Seems the API is pretty small and it's all related. Why not one module?

Reply:
It is small and you know I love a big file. I guess I was thinking of having two layers of interaction, testbldr/should for simple setups and testbldr/pieces for more customized ones. Don't have strong opinions yet, need to flesh it out alot.